apps/examples/ble_rmc/ble_rmc_main.c

```diff
@@ -382,17 +495,58 @@ int ble_rmc_main(int argc, char *argv[])
 		RMC_LOG(RMC_CLIENT_TAG, "disconnect ok\n");
 	}
 
+	if (strncmp(argv[1], "stop", 5) == 0) {
+		if (argc == 3 && strncmp(argv[2], "auto", 5) == 0) {
```

Comment:
How about checking all conditions in one `if` context, line:498 if there is only "auto" for "stop" command?

Reply:
I will add other commands for test like below,
- ble_rmc stop auto
- ble_rmc stop connecting
- ble_rmc stop adv

So, I think this format would be ok.